libs/auth/src/certs.ts

```diff
@@ -224,14 +301,20 @@ export function constructCardCertSubjectWithoutJurisdictionAndCardType(): string
  * Constructs a VotingWorks machine cert subject that can be passed to an openssl command
  */
 export function constructMachineCertSubject(
-  component: Exclude<CustomCertFields['component'], 'card'>,
-  jurisdiction: string
+  machineType: MachineType,
+  jurisdiction?: string
 ): string {
+  assert(
+    (machineType === 'admin' && jurisdiction !== undefined) ||
+      (machineType !== 'admin' && jurisdiction === undefined)
+  );
```

Comment:
I was previously only using this function to create the dev VxAdmin machine cert. I'll now be using this function to create all prod machine certs, hence why I'm just now catching the flawed assumption about jurisdiction always being present on machine certs (it's only present on VxAdmin certs).